web/src/app/main/components/SideBarDrawerList.js

```diff
@@ -134,92 +125,83 @@ export default class SideBarDrawerList extends React.PureComponent {
-                cfg.title,
-                idx,
-              )
-            })}
-          <RequireConfig isAdmin>
-            <Divider aria-hidden />
-            {this.renderAdmin()}
-          </RequireConfig>
-
-          <Divider aria-hidden />
-          {this.renderSidebarNavLink(WizardIcon, '/wizard', 'Wizard')}
-          <Config>
-            {(cfg) =>
-              cfg['Feedback.Enable'] &&
-              this.renderFeedback(
-                cfg['Feedback.OverrideURL'] ||
-                  'https://www.surveygizmo.com/s3/4106900/GoAlert-Feedback',
+  return (
+    <React.Fragment>
+      <div aria-hidden className={classes.logoDiv}>
+        <img
+          className={classes.logo}
+          height={32}
+          src={logo}
+          alt='GoAlert Logo'
+        />
+      </div>
+      <Divider />
+      <List role='navigation' className={classes.list} data-cy='nav-list'>
+        {routeConfig
+          .filter((cfg) => cfg.nav !== false)
+          .map((cfg, idx) => {
+            if (cfg.subRoutes) {
+              return (
+                <NavSubMenu
+                  key={idx}
+                  parentIcon={navIcons[cfg.title]}
+                  parentTitle={cfg.title}
+                  path={getPath(cfg)}
+                  subMenuRoutes={cfg.subRoutes}
+                >
+                  {renderSidebarItem(navIcons[cfg.title], cfg.title)}
+                </NavSubMenu>
               )
             }
-          </Config>
-          {this.renderSidebarLink(
-            LogoutIcon,
-            '/api/v2/identity/logout',
-            'Logout',
-            {
-              onClick: (e) => {
-                e.preventDefault()
-                this.props.logout()
-              },
-            },
-          )}
-          {this.renderSidebarNavLink(CurrentUserAvatar, '/profile', 'Profile')}
-        </List>
-      </React.Fragment>
-    )
-  }
+            return renderSidebarNavLink(
+              navIcons[cfg.title],
+              getPath(cfg),
+              cfg.title,
+              idx,
+            )
+          })}
+        <RequireConfig isAdmin>
+          <Divider aria-hidden />
+          {renderAdmin()}
+        </RequireConfig>
+
+        <Divider aria-hidden />
+        {renderSidebarNavLink(WizardIcon, '/wizard', 'Wizard')}
+        <Config>
+          {(cfg) =>
+            cfg['Feedback.Enable'] &&
+            renderFeedback(
+              cfg['Feedback.OverrideURL'] ||
+                'https://www.surveygizmo.com/s3/4106900/GoAlert-Feedback',
+            )
+          }
+        </Config>
+        {renderSidebarLink(LogoutIcon, '/api/v2/identity/logout', 'Logout', {
+          onClick: (e) => {
+            e.preventDefault()
+            logout()
+          },
+        })}
+        {renderSidebarNavLink(CurrentUserAvatar, '/profile', 'Profile')}
+      </List>
+    </React.Fragment>
+  )
+}
+
+SideBarDrawerList.propTypes = {
+  onWizard: p.func.isRequired,
+  classes: p.object.isRequired,
```

Comment:
Seeing an error in console because classes is required but not provided, could probably remove from propTypes.

Reply:
^ yeah it can be removed, `useStyles` replaces the need for classes to be passed as a prop